sdks/python/apache_beam/io/bigquery.py

```diff
@@ -828,9 +829,20 @@ def convert_row_to_dict(self, row, schema):
       elif field.type == 'FLOAT':
         value = float(value)
       elif field.type == 'TIMESTAMP':
-        value = float(value)
+        # The UTC should come from the timezone library but this is a known
+        # issue in python 2.7 so we'll just hardcode it as we're reading using
+        # utcfromtimestamp. This is just to match the output from the dataflow
```

Comment:
"This is just to match the output from the dataflow runner with the local runner." -> "This is to produce the timestamp string in proper ISO format."